README.md

```diff
@@ -507,6 +507,10 @@ We propose the following operators in addition to the `Observable` interface:
 - `finally()`
   - Like `Promise.finally()`, it takes a callback which gets fired after the
     observable completes in any way (`complete()`/`error()`)
+- `flatMap()`
```

Comment:
I don't think this should appear both in this "exceptional" list, _and_ in the list below about iterator helpers. Seems like it should just exist in the list of iterator helpers, with a note that the semantics are a little different (as you provided below).